src/extension/intellisenseHelper.ts

```diff
@@ -30,9 +43,12 @@ export class IntellisenseHelper {
         // The actions taken in the promise chain below may result in requring a restart.
```

Comment:
This is the main point of entry for enabling Salsa intellisense, so we can just wrap the entire method. i.e.
```
public static setupReactNativeIntellisense(): Q.Promise<void> {
    if (!IntellisenseHelper.isSalsaSupported()) {
        // Do our configuration logic.
        return Q.all([configureWorkspace, configureTypescript]).then(() => { });
    }
    return Q<void>(void 0);
}
```